README.md

```diff
@@ -109,7 +109,7 @@ Set up your Silex application something like this:
     $app['swiftmailer.options'] = array();
 
     // Database config. See http://silex.sensiolabs.org/doc/providers/doctrine.html
-    $app['db.options'] = array(
+    $app['db.options'] => array(
```

Comment:
Hey @WouterSioen, this was actually correct the way it was before. You were probably confused by the following (I marked the line in question with a comment):

```php
$app = array(
    'db.options' => array(   // the arrow would be correct here
        'driver' => 'pdo_mysql',
        'host' => 'localhost',
        'dbname' => 'mydbname',
        'user' => 'mydbuser',
        'password' => 'mydbpassword',
    ),
);
```